### ISSUE TITLE
View Governance Initiated Transfer Status

### ISSUE THREAD

**Issue by pennyandrews** [vegaprotocol/frontend-monorepo#4306]
```[tasklist]
### Tasks
- [ ] Add a draft title or issue reference here
```
## Story

As a user
I want to be able to follow a link to Block Explorer from the Governance Transfer Proposal on the Governance dapp to view the transfer status
So that I can understand the impact of the proposal on the network

## Acceptance Criteria

- [x] I can view a single transfer in explorer (whether initiated via governance or not)
- [x] I can view the status of that transfer on Block Explorer (pending, complete, cancelled) 
- [x] ~I can view the transfer(s) made, if and when it/they happens  on Block Explorer  i.e. date and transfer amount~
- [ ] I can navigate to the transfer view in Explorer from a link in Governance on the proposal page

NB. Removed AC about viewing the actual payments because this implies querying the ledger entries and the ledger entry API is currently FUBAR. Also I think currently the [API](https://docs.vega.xyz/testnet/api/rest/data-v2/trading-data-service-list-ledger-entries) does not support filtering by transfer ID, which I think would make this impossible.  Worth confirming on that and then if we think this is important we can put a request in to Core. 

## Tasks

- [ ] UX (if needed)
- [ ] Design (if needed)
- [ ] Team and stakeholder review
- [ ] Specs reviewed and created or adjusted
- [ ] Implementation
- [ ] Testing (unit and/or e2e)
- [ ] Code review
- [ ] QA review

## Sketch

Just need a link to here 

![Image](https://github.com/vegaprotocol/frontend-monorepo/assets/89918817/0b311648-fce9-4836-926e-e09c4c7a8a7c)

## Additional details / background info

Core API to retrieve single transfer ID is now in scope: https://github.com/vegaprotocol/vega/issues/7461 (THIS IS NOW DONE)

This may be already covered by work over [here](https://github.com/vegaprotocol/frontend-monorepo/issues/4110).

In theory all this requires is that we have a link to a transfer ID in the governance proposal.  Our assumtion is that Core use the proposal ID to create the transfer ID, so we will be ok here but just need to verify.

**Comment by JonRay15** [comment]
Moving to Review ... because if the linked ticket is done right this SHOULD already be covered.

**Comment by JonRay15** [comment]
Although is this blocked by the fact we dont have an API to fetch a single transfer by its ID?  If so we are stuck here.

**Comment by JonRay15** [comment]
Core ticket to fetch transfer by ID is now merged and should be on staging soon.

**Comment by edd** [comment]
Blocked, API ticket created: https://github.com/vegaprotocol/vega/issues/10102

**Comment by JonRay15** [comment]
I dont think that ledger ticket blocks this ticket...

All this ticket is saying is to take the proposal ID from Governance and fire that into getTransfer and pull back the details in block explorer ... no need to use ledger entries unless we want to show the cash flows, which is on another ticket (and not prioritised).
